tests-performance/README.md

```diff
@@ -3,51 +3,43 @@
 #### Structure
 
 1. `automata/`: short database of approximately 70 automata of various complexity. The automata come from different
-  benchmarks. `easiest` benchmarks should take less then second, while, `hardest` can take over 60 seconds.
-
-2. `bin/`: target folder for compiling binaries used for profiling or performance testing.
-3. `src/`: source codes that will be compiled to `bin` directory. The directory contains several templates:
+  benchmarks. `easiest` benchmarks should take less than second, while, `hardest` can take over 60 seconds.
+2. `src/`: source codes that will be compiled to `build` directory. The directory contains several templates that
+   one can use to build ones own benchmarks:
   * `template.cc` contains basic template for arbitrary code to be profiled or performance tested.
-  * `template-with-cli.args.cc` contains basic template that takes file as an command line argument, parses the automaton
+  * `template-with-cli.args.cc` contains basic template that takes file as a command line argument, parses the automaton
      and computes minterms.
   * `template-with-list-of-automata.cc` contains basic template that without arguments takes a hardcoded list of input 
      automata taken from `automata` directory; in case it is run with arguments, it reads the input files that contains
      paths to source automata.
-4. `pycobench`: a performance testing script
-5. `pyco_proc`: a parser of performance results
-6. `scripts/`: several helper scripts for profiling
+3. `pycobench`: a performance testing script
+4. `pyco_proc`: a parser of performance results
+5. `scripts/`: several helper scripts for profiling
   * `scripts/run_callgrind.sh` to print top functions (exclusive time, greater than 1%)
   * `scripts/run_massif.sh` to print memory peak in [B]
-7. `jobs` definitions of jobs for `pycobench`
-8. `inputs` definitions of inputs for `pycobench`
+6. `jobs` definitions of jobs for `pycobench`
+7. `inputs` definitions of inputs for `pycobench`
 
 #### How to profile
 
-1. Copy one of the `template*.cc` files with boilerplate code.
-2. Find comment `HERE COMES YOUR CODE THAT YOU WANT TO PROFILE` and fill your code.
-3. Run `make` from `src` or compile your binary.
- * By default, `Makefile` looks up source build dir (either `../../build`, `../../cmake-build-debug` or `../../cmake-build-release`).
- Run `make BUILD_DIR="..."` to pass different build directory.
+1. Make a copy one of the `template*.cc` files with boilerplate code to start (e.g. `cp src/template.cc src/myprofiling.cc`).
+2. Find comment `HERE COMES YOUR CODE THAT YOU WANT TO PROFILE` and fill it your code.
+3. Run `make release` from root directory or build using `cmake`.
```

Comment:
1. Probably better to add an example on how to run the CMake compilation.
2. And replace the `Run` at the beginning of the step with `compile using` or `build using`. Run is a reserved word when you are talking about tests and perf tests ;)

(The same goes for the section `How to perf test`.)

Reply:
I was thinking about this, and I will not mention the cmake at all. One should build it using the topmost cmake, not standalone (also the CMakeList.txt would have to be more complex), since usually you will make changes to libmata anyway.